core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -277,7 +278,7 @@ public void processJournalEntry(JournalEntry entry) throws IOException {
         long fileId = ((AsyncPersistRequestEntry) innerEntry).getFileId();
         scheduleAsyncPersistenceInternal(getPath(fileId));
       } catch (AlluxioException e) {
-        throw new RuntimeException(e);
+        LOG.error(e.getMessage());
```

Comment:
Should you add a comment to the code saying this? Or a TODO to better handle how async is journaled/replayed?